### ISSUE TITLE
Ruby 3.3.0 Support

### ISSUE THREAD

**Issue by firecall** [Studiosity/grover#212]
Hi, 

Just flagging this, and wondering if an update is in the pipeline for Ruby 3.3.0 support?

Thanks!

```
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    grover (~> 1.1.5) was resolved to 1.1.5, which depends on
      Ruby (< 3.3.0, >= 2.7.0)

  Current Ruby version:
    Ruby (= 3.3.0)
```

**Comment by breisig** [comment]
It is VERY important to get Ruby 3.3.0 support. It's a blocker for us.

**Comment by Laykou** [comment]
The changes are not yet merged, but I was able to just checkout from the branch directly in `Gemfile`:

```
gem 'grover', github: 'sharang-d/grover', branch: 'ruby-3-3'
````

**Comment by abrom** [comment]
Released in [v1.1.6](https://rubygems.org/gems/grover/versions/1.1.6)
